### DISCUSSION TITLE
Disable or update GitHub releases

### DISCUSSION THREAD

**Discussion by appinteractive** [tinacms/tinacms#3877] [General]
Currently when one checks out the Tina repo, gets confused by the latest release that's two years old. 

Either hide the releases in GitHub or update the ci pipeline to populate them with the latest builds and changelogs. 

This will build more trust in the library.

**Comment by appinteractive** [comment]
I just see that the changelog is outdated as well. At least that one should be up to date.

**Reply by logan-anderson** [reply]
@appinteractive This is good feedback we should probably delete the root changelog or put links to the other changelogs in it.

The changelogs can be found in the packages directory. Each package has its own changelog. 

See [tinacms package](https://github.com/tinacms/tinacms/blob/main/packages/tinacms/CHANGELOG.md) as an example.

**Reply by appinteractive** [reply]
Yes I discovered it by digging later but seeing that outdated changelog and releases does confuse quite a bit on first context.
